simple_history/utils.py

```diff
@@ -123,3 +125,19 @@ def bulk_update_with_history(
             default_user=default_user,
             default_change_reason=default_change_reason,
         )
+
+
+def get_change_reason_from_object(obj):
+    if hasattr(obj, "_change_reason"):
+        return getattr(obj, "_change_reason")
+
+    if hasattr(obj, "changeReason"):
+        warning_msg = (
+            "Using the attr changeReason to populate history_change_reason is"
+            " deprecated in 2.10.0 and will be removed in 3.0.0. Use "
+            "_change_reason instead. "
+        )
+        warnings.warn(warning_msg, DeprecationWarning)
```

Comment:
warnings most of the time need a `stacklevel` parameter (usually with value 2, but depends on the calling pattern) so that the message shows the calling code rather that the line with warnings.warn.

Reply:
didn't know that. thanks for the feedback